community-recommendations/Autistica_Turing Citizen Science Platform (Responses) - Form Responses 1.csv

```diff
@@ -0,0 +1,51 @@
+What do you expect from this session?
+Do you require any additional support?
+What's your preferred style of working within a group? Written, verbal, images? - Catering to accessibility
+Incorporate a quiet space if it gets too much
+Do people have mobility impairments? Be prepared for this.
+No more than 2-3 questions per turn (break up session) - if people have extra questions or suggestions put in a comment box so they can still be responded to
+In case people have difficulties, e.g. anxiety, have someone to assist/make sure they get included.
+Make clear that people can't make their own recordings.
+Booklet or material about what to expect.",I am autistic,Yes
+9/15/2019 19:25:44,Yes,,I am a parent/ guardian of someone autistic,Yes
+9/20/2019 16:07:11,Yes,What can the citizen science platform offer in terms of accessibility and flexibility?,I am autistic,Yes
+9/24/2019 13:51:45,Yes,,"I am autistic, I am a parent/ guardian of someone autistic, I know someone who is autistic",Yes
+9/24/2019 14:15:53,Yes,"I am concerned about the difficulties travelling in / navigating confined spaces like tube trains and lifts. On 17 th September l attended a council meeting in a new mixed use building that has been passed as finished yet the lift l exited from the second floor to ground level had ply board fascias next to the metal control panel. There was lots of striped sticky tape holding the boards in place and l felt very uncomfortable.  Could they collapse and fall in on me or if there was an electric spark would they ignite. Where was the panic button. Nowhere in plain sight. I’m travelling in a FIRE BOX. I was shaking inside the whole journey down. I complained at the reception desk as l signed out, asking hey they didn’t see it as a priority to finish it and have emergency signage clearly visible. I asked them to log the complaint and emailed my council too. I attended a meeting on 23 rd. The lift is finished. I feel my feelings were validated.","I am autistic, I am a parent/ guardian of someone autistic, I know someone who is autistic",Yes
+9/24/2019 15:01:40,Yes,I think this question could include more specific examples. ,"Someone in my family is autistic, I know someone who is autistic",Yes
+10/3/2019 23:54:28,Yes,,"I am autistic, I am a parent/ guardian of someone autistic",Yes
+11/6/2019 18:44:51,Yes,"I just want to be able to do more than exist.  I expend so much energy trying to survive my environment and control myself ro be ""normal"" that I have no energy to do anything else.  I travel to work, I work, I eat, I travel home, I eat, I sleep.  I don't hang out with friends or read new books or binge watch because I simply don't have the energy or concentration left after battling through the day.  The older I get, the worse it gets.  I would just like to be able to *live*",I am autistic,Yes
+11/7/2019 15:27:03,Yes,,I am autistic,Yes
+11/7/2019 15:30:17,Yes,"Travel Blog as discussed on Twitter (@xmorpheus)
```

Comment:
This line contains a Twitter handle, does this meet your anonymisation criteria?

Reply:
Thanks for your review @sgibson91 I agree - I've removed the twitter handle from the csv and I'll add that for now and then regenerate the pdf. :)

Georgia